apps/web-giddh/src/app/settings/integration/setting.integration.component.html

```diff
@@ -397,29 +398,106 @@ <h2 style="float: left; font-size: 20px">ICICI Bank</h2>
+                                </div>
+                                <p class="error-msg text-right"></p>
+                            </div>
+                            <div class="form-group col-sm-3 col-md-2" *ngIf="item.maxBankLimit !== 'max'">
+                                <ng-container *ngIf="index===0"> <label>Amount Upto <span class="required">*</span></label>
+                                    <br></ng-container>
+                                <div class="d-flex">
+                                    <div class="mr-05 custom-select select-caret pos-rel">
+                                        <sh-select [options]="amountUpToList" [(ngModel)]="item.maxBankLimit" (selected)="selectedMaxOrCustom(index, true, ind);" (ngModelChange)="maxLimitOrCustomChanged($event, index , true, ind)" name="maxBankLimit{{ind}}{{index}}">
+                                        </sh-select>
+                                    </div>
+                                    <div class="ml-05">
+                                        <input type="number" class="form-control" required [(ngModel)]="item.amount" name="amount{{ind}}{{index}}" placeholder="Amount">
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="form-group col-sm-3 col-md-2">
+                                <ng-container *ngIf="index===0"> <label class="d-flex relative info-icon">Select OTP <span class="required">*</span>
+                                <i class="icon-info cursor-pointer" tooltip="Content here.."></i>
+                            </label></ng-container>
+                                <div class="custom-select select-caret pos-rel">
+                                    <sh-select [options]="typeOTPList" [(ngModel)]="item.otpType" required name="otpType{{ind}}{{index}}">
+                                    </sh-select>
+                                </div>
+                            </div>
+                            <div class="form-group col-sm-3 col-md-2">
+                                <ng-container *ngIf="index===0"> <label class="d-flex relative info-icon">Approval Name <span class="required">*</span>
+                                <i class="icon-info cursor-pointer"  tooltip="Content here.."></i>
+                            </label></ng-container>
+                                <div class="custom-select select-caret pos-rel" *ngIf="item.otpType!=='BANK'">
+                                    <sh-select [options]="approvalNameList" class="text-left" [(ngModel)]="item.approvalUniqueName" name="approvalUniqueName{{ind}}{{index}}" required [placeholder]="'Select Bank'" [multiple]="false">
+                                        <ng-template #optionTemplate let-option="option">
+                                            <a href="javascript:void(0)" *ngIf="option?.additional[0].mobileVerified" class="list-item relative">
```

Comment:
Consider using safe navigation
```suggestion
                                            <a href="javascript:void(0)" *ngIf="option?.additional[0]?.mobileVerified" class="list-item relative">
```